website/docs/r/ip_pool.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "appgate"
+page_title: "APPGATE: appgate_ip_pool"
+sidebar_current: "docs-appgate-resource-ip_pool"
+description: |-
+   Create a new IP Pool.
+---
+
+# appgate_ip_pool
+
+Create a new IP Pool..
+
+## Example Usage
+
+```hcl
+
+resource "appgate_ip_pool" "example_ip_pool" {
+  name            = "ip range example"
+  lease_time_days = 5
+  ranges {
+    first = "10.0.0.1"
+    last  = "10.0.0.254"
+  }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `ip_version6`: (Optional) Whether the IP pool is for v4 or v6.
```

Comment:
What happens if ip_version6 is not set. Do we infer the version from the actual ips or just default to ipv4?

Reply:
it defaults to ipv4